skyportal/models.py

```diff
@@ -453,6 +466,9 @@ class Photometry(Base):
     ra_unc = sa.Column(sa.Float, doc="Uncertainty of ra position [arcsec]")
     dec_unc = sa.Column(sa.Float, doc="Uncertainty of dec position [arcsec]")
 
+    detected = sa.Column(sa.Boolean, default=True, nullable=False,
```

Comment:
So anything with a `candid` in ZTF speak?

Reply:
Yeah. This may not be the best way to implement this. But currently I am leaving it to the user to define if something counts as a detection, rather than enforcing specific criteria. See convo in #531

Reply:
I decided to define `detected` as 5-sigma